src/utils/permalinks/Permalinks.js

```diff
@@ -332,6 +332,10 @@ export function tryTransformPermalinkToLocalHref(permalink: string): string {
             if (permalinkParts.roomIdOrAlias) {
                 const eventIdPart = permalinkParts.eventId ? `/${permalinkParts.eventId}` : '';
                 permalink = `#/room/${permalinkParts.roomIdOrAlias}${eventIdPart}`;
+                if (permalinkParts.viaServers.length > 0) {
+                    const riotPermalinkConstructor = new SpecPermalinkConstructor();
```

Comment:
the variable name is wrong

Reply:
oh yeah, was gonna use the other c'tor but it needed a URL and the method I care about is identical in both